src/app/lib/config/services/afaanoromoo.js

```diff
@@ -184,6 +184,9 @@ export const service = {
       hasRadioSchedule: true,
       onFrontPage: false,
     },
+    recommendations: {
+      hasRecommendations: false,
```

Comment:
```suggestion
      hasStoryRecommendations: false,
```

I think we agreeed to use the key `hasStoryRecommendations` in refinement to make it clear this refers to legacy CPS Story Recommendations as opposed to any new recommendations implemented later for articles etc.

Reply:
🙈